src/orion/benchmark/task/profet/fcnet.py

```diff
@@ -0,0 +1,69 @@
+""" Simulated Task consisting in training a fully-connected network. 
+
+The task is created using the Profet algorithm:
+
+```
+@misc{klein2019metasurrogate,
```

Comment:
Have you tried running sphinx to see how this gets formatted? Otherwise I would use a standard citation style like _chicago_.

Reply:
No I haven't, but I have no clue how to do this. Can this wait for another PR?

Reply:
That's the kind of boring stuff that may never be done if delayed. :) 

You could use this

```
For more information on Profet, see original paper at https://arxiv.org/abs/1905.12982.

Klein, Aaron, Zhenwen Dai, Frank Hutter, Neil Lawrence, and Javier Gonzalez. "Meta-surrogate benchmarking for 
hyperparameter optimization." Advances in Neural Information Processing Systems 32 (2019): 6270-6280.
```

Reply:
Updated the citation style for all the profet tasks, based on what you proposed.

Reply:
Also removed it from the header of the individual tasks, and into the __init__.py for the tasks/profet package.

Reply:
Great, thanks!